src/readbin/symbols.ml

```diff
@@ -66,4 +66,20 @@ let read ?demangle ~filename arch base  : string table =
             let mem = Memory.view ~from ~words base |> ok_exn in
             let name = demangle name in
             Table.add syms mem name |> ok_exn
-          with exn -> syms))
+          with _exn -> syms))
+
+let read_addrset filename : Addr.Set.t =
+  Addr.Set.t_of_sexp @@ Sexp.load_sexp filename
+
+let write_addrset ?filename (addr_set : Addr.Set.t) : unit =
```

Comment:
This comment is not addressed. I still insist on rewriting all functions in this module, to accept `out_channel`, without any `?filename` arguments. This is a bad design. And you proved this yourself, by accidentally closing `stdin` and thus breaking the system. Moreover, you hide the user interface logic under the library abstraction (without even providing comments).